src/index.js

```diff
@@ -0,0 +1,111 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const AWS = require('aws-sdk');
+
+const CONCLUSIONS = [
+	'neutral',
+	'skipped',
+	'success',
+	'cancelled',
+	'timed_out',
+	'action_required',
+	'failure',
+];
+
+// Fetch all the workflow run jobs
+const getJobs = async context => {
+	const RUN_ID = core.getInput('RUN_ID');
```

Comment:
I think it is possible to get the run id from the context `github.run_id`, this way we wouldn't need the RUN_ID as input.

Reply:
Same as the token, this is now taken from the `github` object being input as `GITHUB_CONTEXT`